# Instructions for input data and code
1. Fig.1: pcoa.R: R scripts used for the pcoa analysisï¼›

2. Fig.2: Kitrinoviricotardrp.fasta, Leviviridaerdrp.fasta and Pisuviricotardrp.fasta: sequence data used to generate RdRp-based phylogenetic tree using the iToL.

3. Protein alignment consensus tree.newick: data used to generate RdRp domain sequences in fig.2b using Geneious v8.1.9 (https://www.geneious.com).

4. rdrpref.fasta:  RDRP reference sequences used in this study from Zayed et al. (2022) and Wolf et al. (2020).

5. Fig.3: host in different depth.xlsx: the data used to generate fig.3c; 

6. circlize.R: R scripts used for fig.3c.

7. Fig.4: viral kroA sequence.fasta and viral nasA sequence.fasta were the sequences used in fig.4;

8. genemap-R: R scripts used for the gene map;

9. genemap-C-virus.txt: data used in the fig.4.
